bin/opm

```diff
@@ -1345,7 +1345,7 @@ sub install_target ($$) {
         shell "rm -rf '$dist_dir'";
     }
 
-    shell "tar -xzf '$dist_file'";
+    shell "tar --no-same-owner -xzf '$dist_file'";
```

Comment:
@jkeys089 Never mind, I messed up the order of the options. Seems that it's working in OS X :)